src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The truck has just started working");

```

Comment:
The message in the 'doWork' method should be more informative and indicate the specific type of Machine that is working. For example, 'Truck is working.' would be more informative.